lib/mogrify.ex

```diff
@@ -54,12 +54,13 @@ defmodule Mogrify do
 
   iex> open("test/fixtures/rbgw.png") |> histogram
   [
-    %{"blue" => 255, "count" => 400, "green" => 0, "hex" => "#0000ff", "red" => 0},
-    %{"blue" => 0, "count" => 225, "green" => 255, "hex" => "#00ff00", "red" => 0},
-    %{"blue" => 0, "count" => 525, "green" => 0, "hex" => "#ff0000", "red" => 255},
-    %{"blue" => 255, "count" => 1350, "green" => 255, "hex" => "#ffffff", "red" => 255}
+    %{"alpha" => 0, "blue" => 255, "count" => 400, "green" => 0, "hex" => "#0000ff", "red" => 0},
```

Comment:
I think it makes sense for `alpha` to default to 255 rather than 0 if it's RGB data.